src/App.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import './App.css';
+import Header from './components/public/Header';
+import LandingPage from './components/public/LandingPage';
```

Comment:
Here too you can create another folder named "pages" and move the LandingPage component inside it.